packages/react-components/src/icons/card-view.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+/* istanbul ignore file */
+
+const cardViewIcon = (
+  <svg
+    width={24}
+    height={24}
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    stroke="#4D646B"
```

Comment:
shiet just realized we probably have a lot of color duplication in these icons. meh, can probably do one pass replacing them with the defined colors at some point